app/controllers/conference_registrations_controller.rb

```diff
@@ -20,6 +20,11 @@ def new
       redirect_to edit_conference_conference_registrations_path(@conference.short_title)
     end
 
+    if @conference.registration_limit_exceeded?
```

Comment:
Ok, I have added the validation in conference model and added a msg to the user in _registration in conference show when the limit has exceeded. 

If I miss anything let me know :)